src/filter/filters.ts

```diff
@@ -166,6 +178,39 @@ export const handleAbiDecode = (context: any, filter: { $abi: Abi }) => {
   }
 }
 
+export const handleAbstractAbiDecode = (
+  context: any,
+  filter: { $abiAbstract?: Abi },
+) => {
+  const decodedReturn: ReturnType<typeof handleAbiDecode>[] = []
+  const elementCount = filter.$abiAbstract!.length
+  const contextMap = new Map<string, number>()
+  for (let i = 0; i < context.length - 6; i++) {
+    contextMap.set(context.substring(i, i + 6), i)
+  }
+  for (let i = 0; i < elementCount; i++) {
+    const abiItem = filter.$abiAbstract![i]
+    if (abiItem.type === 'function') {
+      const functionSelector = getFunctionSelector(abiItem)
+      const functionSelectorSubstring = functionSelector.substring(2)
```

Comment:
also a comment here why it has to be 2. for context, so if someone has to debug the code that could understand

Reply:
Yep can do 😊

Reply:
send itttt